### PR TITLE
fix #55984 t.pit.jp

### LINKED ISSUE #55984: t.pia.jp
### Issue URL (Annoyance)
 [https://t.pia.jp/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ft.pia.jp%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | And 10
AdGuard version: | 3.5 nightly 1 (5)
Browser: | Chrome (data compression)
AdGuard mode: | VPN
Filtering quality: | High-quality
HTTPS filtering: | enabled
DNS requests blocking: | enabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Japanese

### PR DESCRIPTION
**Pull request by Yuki2718**
https://github.com/AdguardTeam/AdguardFilters/issues/55984

### CODE REVIEW

File: AnnoyancesFilter/sections/self-promo.txt
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/self-promo.txt:44]: I would add this rule to the popup.txt, but please don't move it now, wait for another review.

Comment (Yuki2718) [reply AnnoyancesFilter/sections/self-promo.txt:44]: All right.

Comment (Alex-302) [reply AnnoyancesFilter/sections/self-promo.txt:44]: self-promo.txt is also proper

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**
